qa/tasks/cephfs/test_quiesce.py

```diff
@@ -559,6 +560,26 @@ def test_quiesce_slow(self):
         self._wait_for_quiesce_complete(reqid)
         self._verify_quiesce(root=self.subvolume)
 
+    def test_quiesce_dir_fragment(self):
+        """
+        That quiesce completes with fragmentation in the background.
+        """
+
+        self.config_set('mds', 'mds_bal_split_size', '10')
+        self.config_set('mds', 'mds_bal_merge_size', '1') # do not merge
+        self.config_set('mds', 'mds_bal_split_bits', '1')
+        self._configure_subvolume()
+        self._client_background_workload()
+
+        # time for the workload to get busy
+        time.sleep(5)
+
+        J = self.fs.rank_tell("quiesce", "path", self.subvolume)
```

Comment:
Should we do the quiesce in a loop along with some checks telling us whether we've hit the wanted flow?

Reply:
Ack